presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoControlUtils.java

```diff
@@ -0,0 +1,182 @@
+        }
+        return Response.status(OK).build();
+    }
+
+    private static void stopHelper(StopType stopType)
+            throws PrestoManagerException
+    {
+        switch (stopType) {
+            case TERMINATE:
+                int prestoTerminate = executeCommand("service", "presto", "stop");
+                if (prestoTerminate != 0) {
+                    throw new PrestoManagerException("Failed to stop presto", prestoTerminate);
+                }
+                break;
+            case KILL:
+                executeCommand("sudo", LAUNCHER_SCRIPT, "kill");
+                int prestoKill = executeCommand("service", "presto", "status");
+                if (prestoKill != 3) {
+                    throw new PrestoManagerException("Failed to stop presto", prestoKill);
+                }
+                break;
+            case GRACEFUL:
+                try {
+                    if (getFileProperty(PRESTO_CONFIG_FILE, "coordinator").equalsIgnoreCase("true")) {
+                        throw new PrestoManagerException("Coordinator can't be gracefully stopped");
+                    }
+                    String prestoPort = getFileProperty(PRESTO_CONFIG_FILE, "http-server.http.port");
+                    Response response = newRequestBuilder(new URI(format("localhost:%s/v1/info/state", prestoPort)))
+                            .withBody(entity((new Gson().toJson("SHUTTING_DOWN")), APPLICATION_JSON_TYPE)).buildPut().invoke();
+                    if (response.getStatus() != 200) {
+                        throw new PrestoManagerException("Failed to stop presto gracefully");
+                    }
+                }
+                catch (IOException | URISyntaxException e) {
+                    throw new PrestoManagerException("Error in stopping presto", e);
+                }
+        }
+    }
+
+    public static Response restartUsingRpm()
+    {
+        try {
+            if (executeCommand("rpm", "-q", "presto-server-rpm") != 0) {
+                LOGGER.debug("Presto is not installed");
+                return Response.status(NOT_FOUND).build();
+            }
+            int prestoRestart = executeCommand("service", "presto", "restart");
+            if (prestoRestart != 0) {
+                throw new PrestoManagerException("Failed to restart presto", prestoRestart);
+            }
+        }
+        catch (PrestoManagerException e) {
+            LOGGER.error(e.getMessage(), e.getCause());
+            return Response.status(INTERNAL_SERVER_ERROR).build();
+        }
+        return Response.status(OK).build();
+    }
+
+    public static Response statusUsingRpm()
+    {
+        try {
+            if (executeCommand("rpm", "-q", "presto-server-rpm") != 0) {
+                LOGGER.debug("Presto is not installed");
+                return Response.status(OK).entity(GSON.toJson("Presto is not installed"))
+                        .type(APPLICATION_JSON_TYPE).build();
```

Comment:
For ```status```  this a correct behavior. 
Successfully retrieved the status and the status is "Presto not installed"